pkg/capacity/topology/mock.go

```diff
@@ -54,6 +54,9 @@ func (mt *Mock) List() []*Segment {
 func (mt *Mock) Run(ctx context.Context) {
 }
 
+func (mt *Mock) RunWorker(ctx context.Context) {
+}
+
```

Comment:
This replaces the `Run` method, doesn't it?

Reply:
Yes, you mean I can remove topology informer `Run` method directly?

Reply:
Yes. I thought that was the proposal: clarify that this is not a `SharedInformer` by naming the "Run" method not "Run". Nothing else calls it, so we are free to choose an arbitrary name.

If that makes the patch very large, then perhaps we should simply stick with `Run`.